src/commands/status/platform.js

```diff
@@ -0,0 +1,176 @@
+const { table } = require('table');
+const fetch = require('node-fetch');
+const chalk = require('chalk');
+
+const BaseCommand = require('../../oclif/command/BaseCommand');
+const CoreService = require('../../core/CoreService');
+
+const ContainerIsNotPresentError = require('../../docker/errors/ContainerIsNotPresentError');
+
+class CoreStatusCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @param {CoreService} coreService
+   * @param {Config} config
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    args,
+    flags,
+    dockerCompose,
+    createRpcClient,
+    config,
+  ) {
+    if (config.options.network === 'testnet') {
+      // eslint-disable-next-line no-console
+      console.log('Platform is not supported on testnet yet!');
+      this.exit();
+    }
+
+    const rows = [];
+
+    const coreService = new CoreService(
+      createRpcClient(
+        {
+          port: config.get('core.rpc.port'),
+          user: config.get('core.rpc.user'),
+          pass: config.get('core.rpc.password'),
+        },
+      ),
+      dockerCompose.docker.getContainer('core'),
+    );
+
+    // Collect core data
+    const {
+      result: {
+        IsSynced: coreIsSynced,
+      },
+    } = await coreService.getRpcClient().mnsync('status');
+
+    // Collecting platform data fails if Tenderdash is waiting for core to sync
+    if (coreIsSynced === false) {
+      // eslint-disable-next-line no-console
+      console.log('Platform status is not available until core sync is complete!');
+      this.exit();
+    }
+
+    // Collect platform data
+    const tendermintStatusRes = await fetch(`http://localhost:${config.options.platform.drive.tendermint.rpc.port}/status`);
+    const {
+      result: {
+        node_info: {
+          version: platformVersion,
+          network: platformNetwork,
+        },
+        sync_info: {
+          catching_up: platformCatchingUp,
+          latest_app_hash: platformLatestAppHash,
+          latest_block_height: platformLatestBlockHeight,
+        },
+      },
+    } = await tendermintStatusRes.json();
+
+    const tendermintNetInfoRes = await fetch(`http://localhost:${config.options.platform.drive.tendermint.rpc.port}/net_info`);
+    const {
+      result: {
+        n_peers: platformPeers,
+      },
+    } = await tendermintNetInfoRes.json();
+
+    const explorerBlockHeightRes = await fetch('https://rpc.cloudwheels.net:26657/status');
```

Comment:
Let's check it if we use network evonet.

Reply:
I've fixed this by following the same pattern as in `status:core` to check if an external block explorer exists for the current config before running the query. This condition will currently never be reached, since there is a testnet check/abort at the top of the command and the command can never run for `local` config because core never reaches `IsSynced` status. But this check will still be useful as platform is enabled on more networks.